src/HotChocolate/AspNetCore/src/AspNetCore/Subscriptions/WebSocketSession.cs

```diff
@@ -35,7 +38,8 @@ public class WebSocketSession : ISocketSession
 
     public async Task HandleAsync(CancellationToken cancellationToken)
     {
-        using var cts = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken);
+        using var cts = CancellationTokenSource.CreateLinkedTokenSource(
+            new [] { cancellationToken, _applicationStopping });
```

Comment:
I think there exist a overload of CreateLinkedTokenSource with exactly two parameters :)

Reply:
Are you getting alloc allergies? ;) I'll fix it.

Reply:
done